auxiliary/stata-guide.tex

```diff
@@ -112,7 +112,11 @@ \section{Writing good code}
 
 \section{Using the code examples in this book}
 
-Providing some standardization to Stata code style is also a goal of our team.
+Providing some standardization to Stata code style
+is also a goal of our team.
+Stata is one of several programming languages used at DIME,
+but since it exists fewer good resources based on Stata,
+we decided to use Stata for our examples in this book.
```

Comment:
```suggestion
Providing some standardization for Stata code style
is also a goal of our team.
Stata is one of several programming languages used at DIME,
but since few high-quality resources based on Stata exist
relative to its importance and frequency of use,
we decided to use Stata for the examples in this book.
```